### ISSUE TITLE
Use new Java version strings

### ISSUE THREAD

**Issue by kaspernielsen** [svenruppert/docker-image-generator#2]
Hi, 

Any chance we can get the real Java version strings for never JDKs?
For example, 12.0.1 instead of 1.12.0.1?

Thanks

**Comment by svenruppert** [comment]
Hi @kaspernielsen , sure we could change this.  Main thing here: I am using the strings provided from Jabba. But we could create a **VersionNumberTransformer**.

**Comment by kaspernielsen** [comment]
Yeah, that would be great. It is a bit confusing when the version strings do not match exactly.
